pandas/core/frame.py

```diff
@@ -1943,32 +1943,86 @@ def _sizeof_fmt(num, size_qualifier):
         _put_lines(buf, lines)
 
     def memory_usage(self, index=True, deep=False):
-        """Memory usage of DataFrame columns.
+        """
+        Return the memory usage of each column in bytes.
+
+        The memory usage can optionally include the contribution of
+        the index and elements of `object` dtype.
+
+        A configuration option, `display.memory_usage` (see Parameters)
```

Comment:
There seems to be missing something in this sentence.

Reply:
Fixed.